code/modules/reagents/chemistry/reagents/impure_reagents/impure_medicine_reagents.dm

```diff
@@ -540,3 +544,207 @@
+	. = ..()
+	if(!iscarbon(owner))
+		return
+	var/mob/living/carbon/carbon = owner
+	carbon.dna?.remove_mutation(speech_option)
+
+/datum/reagent/inverse/neurine
+	name = "Neruwhine"
+	description = "Induces a temporary brain trauma in the patient by redirecting neuron activity."
+	color = "#DCDCAA"
+	ph = 13.4
+	addiction_types = list(/datum/addiction/medicine = 8)
+	metabolization_rate = 0.025 * REM
+	tox_damage = 0
+	//The temporary trauma passed to the owner
+	var/datum/brain_trauma/temp_trauma
+
+/datum/reagent/inverse/neurine/on_mob_life(mob/living/carbon/owner, delta_time, times_fired)
+	.=..()
+	if(temp_trauma)
+		return
+	if(!(DT_PROB(creation_purity*10, delta_time)))
+		return
+	var/traumalist = subtypesof(/datum/brain_trauma)
+	traumalist -= /datum/brain_trauma/severe/split_personality //Uses a ghost, I don't want to use a ghost for a temp thing.
+	var/datum/brain_trauma/trauma = pick(traumalist)
+	var/obj/item/organ/brain/brain = owner.getorganslot(ORGAN_SLOT_BRAIN)
+	if(brain.brain_gain_trauma(trauma, TRAUMA_RESILIENCE_MAGIC))
+		temp_trauma = trauma
+
+/datum/reagent/inverse/neurine/on_mob_delete(mob/living/carbon/owner)
+	.=..()
+	if(!temp_trauma)
+		return
+	if(istype(temp_trauma, /datum/brain_trauma/special/imaginary_friend))//Good friends stay by you, no matter what
+		return
+	owner.cure_trauma_type(temp_trauma, resilience = TRAUMA_RESILIENCE_MAGIC)
+
+/datum/reagent/inverse/corazone
+	name = "Corazargh" //It's what you yell! Though, if you've a better name feel free. Also an omage to an older chem
+	description = "Interferes with the body's natural pacemaker, forcing the patient to manually beat their heart."
+	color = "#5F5F5F"
+	self_consuming = TRUE
+	ph = 13.5
+	addiction_types = list(/datum/addiction/medicine = 2.5)
+	metabolization_rate = 0.01 * REM
+	chemical_flags = REAGENT_DEAD_PROCESS
+	tox_damage = 0
+	///The old heart we're swapping for
+	var/obj/item/organ/heart/original_heart
+	///The new heart that's temp added
+	var/obj/item/organ/heart/cursed/manual_heart
+
+///Creates a new cursed heart and puts the old inside of it, then replaces the position of the old
+/datum/reagent/inverse/corazone/on_mob_add(mob/living/owner)
+	if(!iscarbon(owner))
+		return
+	var/mob/living/carbon/carbon_mob = owner
+	original_heart = owner.getorganslot(ORGAN_SLOT_HEART)
+	if(!original_heart)
+		return
+	manual_heart = new(null, src)
+	original_heart.Remove(carbon_mob, special = TRUE) //So we don't suddenly die
+	original_heart.forceMove(manual_heart)
+	original_heart.organ_flags |= ORGAN_FROZEN //Not actually frozen, but we want to pause decay
+	manual_heart.Insert(carbon_mob, special = TRUE)
+	//these last so instert doesn't call them
+	RegisterSignal(carbon_mob, COMSIG_CARBON_GAIN_ORGAN, .proc/on_gained_organ)
+	RegisterSignal(carbon_mob, COMSIG_CARBON_LOSE_ORGAN, .proc/on_removed_organ)
+	to_chat(owner, "<span class='warning'>You feel your heart suddenly stop beating on it's own - you'll have to manually beat it!</spans>")
```

Comment:
```suggestion
	to_chat(owner, "<span class='warning'>You feel your heart suddenly stop beating on its own - you'll have to manually beat it!</spans>")
```

Reply:
also, you might want a check or something for if the patient doesn't need a heart, due to their species, that one bioware surgery, etc.